fe/fe-core/src/test/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessorTest.java

```diff
@@ -1155,6 +1165,22 @@ public void testRefreshPartitionWithMulParColumnsHiveTable2() throws Exception {
         Assert.assertTrue(plan.contains("partitions=3/7"));
     }
 
+    @Test
+    public void testRangePartitionRefreshWithJDBCTable() throws Exception {
+        Database testDb = GlobalStateMgr.getCurrentState().getDb("test");
+        MaterializedView materializedView = ((MaterializedView) testDb.getTable("jdbc_parttbl_mv"));
+        HashMap<String, String> taskRunProperties = new HashMap<>();
+        taskRunProperties.put(PARTITION_START, "20230801");
+        taskRunProperties.put(TaskRun.PARTITION_END, "20230805");
+        taskRunProperties.put(TaskRun.FORCE, Boolean.toString(false));
+        Task task = TaskBuilder.buildMvTask(materializedView, testDb.getFullName());
+        TaskRun taskRun = TaskRunBuilder.newBuilder(task).properties(taskRunProperties).build();
+        taskRun.initStatus(UUIDUtil.genUUID().toString(), System.currentTimeMillis());
+        taskRun.executeTaskRun();
+        Collection<Partition> partitions = materializedView.getPartitions();
+        Assert.assertEquals(5, partitions.size());
```

Comment:
Can you add more cases to verify the materialized view can incremently be refreshed by the incremental partitions of the base jdbc table?

like `testHivePartitionPruneNonRefBaseTable3`?

Reply:
I'm trying to add this case

Reply:
Add cases to verify the materialized view can incremently be refreshed by the incremental partitions of the base jdbc table.